frameworks/hdfs/docs/security.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt:
+title: Security
+menuWeight: 22
+---
+
+# DC/OS Apache HDFS Security
+
+The DC/OS Apache HDFS service supports HDFS's native transport encryption, authentication, and authorization mechanisms. The service provides automation and orchestration to simplify the usage of these important features.
+
+A good overview of these features can be found [here](https://hadoop.apache.org/docs/r2.7.2/hadoop-project-dist/hadoop-common/SecureMode.html).
+
+*Note*: These security features are only available on DC/OS Enterprise 1.10 and above.
+
+## Transport Encryption
+
+With transport encryption enabled, DC/OS Apache HDFS will automatically deploy all nodes with the correct configuration to encrypt communication via TLS. The nodes will communicate securely between themselves using TLS. Optionally, plaintext communication can be left open to clients.
+
+The service uses the [DC/OS CA](https://docs.mesosphere.com/latest/security/ent/tls-ssl/) to generate the SSL artifacts that it uses to secure the service. Any client that trusts the DC/OS CA will consider the service's certificates valid.
+
+*Note*: Enabling transport encryption is _required_ to use [SSL authentication](#ssl-authentication) for [authentication](#authentication) (authn), but is optional for [Kerberos authn](#kerberos-authn).
+
+{% include services/security-configure-transport-encryption.md
+    techName="Apache HDFS" %}
+
+<!--
+TO BE CONFIRMED
+*Note*: It is possible to update a running DC/OS Apache HDFS service to enable transport encryption after initial installation, but the service may be unavilable during the transition. Additionally, your HDFS clients will need to be reconfigured unless `service.security.transport_encryption.allow_plaintext` is set to true. -->
+
+#### Verify Transport Encryption Enabled
+
+???
```

Comment:
Is this section gonna be filled out for this PR?

Reply:
Whoops.

I'll add something...